run-integration-test/README.md

```diff
@@ -33,8 +33,12 @@ Supported Kubernetes version can be inspected on the official Replicated documen
 - `test-platform`(required, eg: `kind-1.31.2-amd64`)
 - `test-run` (required, `test-suite` or `test`)
 - `test-parameter` (defaults to `smoke`)
+- `test-parallelism` (defaults to `2`)
```

Comment:
note: We want to cut down on the number of action inputs and instead use the integration test config file to configure such options. This enables custom values per operator, because otherwise each operator would use the same value (the integration test workflow is templated).

Reply:
I've removed this as an input and the value is now only read from the config (or set to the default of 2).